rc4airconbotmvp.py

```diff
@@ -111,42 +97,38 @@ def start(update, context):
 def prompt_id(update, context):
```

Comment:
Shift the previous line to before the `start` method (because `start` uses its value)
```
UPDATE_ID, UPDATE_NOTIF, UPDATE_END = range(3)
```